src/test/e2e/PageObjects/AdminDashboard.page.ts

```diff
@@ -0,0 +1,15 @@
+import { CommonPage } from './Common.page';
+import { ManualUploadPage } from './ManualUpload.page';
+
+const helpers = require('../Helpers/Selectors');
+
+export class AdminDashboardPage extends CommonPage {
+  async clickUploadFileCard(): Promise<ManualUploadPage> {
+    await $(helpers.UploadFile).catch(() => {
+      console.log(`${helpers.EmailSubscriptionLink} not found`);
```

Comment:
Think this needs updating from EmailSubscriptionLink

Reply:
^^

Reply:
Whoops, should be fixed now